src/android/java/com/pspdfkit/cordova/PSPDFKitCordovaPlugin.java

```diff
@@ -170,6 +173,8 @@
                     else throw new IllegalArgumentException(String.format("Invalid search type: %s", value));
                 } else if ("autosaveEnabled".equals(option)) {
                     builder.autosaveEnabled(options.getBoolean("autosaveEnabled"));
+                } else if ("secret".equals(option)) {
```

Comment:
Please rename this to `password` so it stays conform with the PSPDFKit parameter name, and move the extraction of this argument up one level into the `exec()` method (since it is not part of the configuration object).